desktop/apps/artwork/components/artists/market_insights.js

```diff
@@ -0,0 +1,22 @@
+import $ from 'jquery'
+
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { Contents } from '@artsy/reaction-force/dist/Components/Artist/MarketInsights'
+import { ContextProvider } from '@artsy/reaction-force/dist/Components/Artsy'
+
+const renderMarketInsightsFor = (artistID) => {
+  const elementID = 'market-insights-container-' + artistID
+  ReactDOM.render((
+    <ContextProvider>
+      <Contents artistID={artistID} />
+    </ContextProvider>), document.getElementById(elementID))
+}
+
+function setupMarketInsights () {
+  $('section.artwork-section.artwork-artist[data-artist-id]').each((i, el) => {
```

Comment:
Loop thru each artist section below the fold, and render market insights in each unique (scoped by id) market insights div